client/gutenberg/extensions/contact-form/editor.js

```diff
@@ -4,7 +4,7 @@
  * Internal dependencies
  */
 import registerJetpackBlock from 'gutenberg/extensions/presets/jetpack/utils/register-jetpack-block';
-import { fields, name, settings } from '.';
+import { children, name, settings } from '.';
```

Comment:
Can we rename this to InnerBlocks instead?

Reply:
I was hoping to maybe re-use the same nomenclature for presets later. But since we're not using them that way yet, fine to rename and re-visit later :+1:

Reply:
Yeah, I think `children` can be a bit confusing. `allowedBlocks` (which mirrors the prop name of `<InnerBlocks />`) or `nestedBlocks` also make sense to me. We can also link to https://github.com/WordPress/gutenberg/tree/master/packages/editor/src/components/inner-blocks for context

Reply:
I wouldn't want to use `allowedBlocks` because we allow more then just the fields to be added to the contact form block.

Reply:
`nestedBlocks` sounds kinda good to me, tho :thinking:

Reply:
Or maybe just `childBlocks`? I mean they do reference their `parent`...

Reply:
3b7b2a7 -- I settled on `childBlocks`. `InnerBlocks` seems to refer more to the specific component used for that, `nestedBlocks` seems to indicate more the abstract concepts (possibly including multiple layers of nesting).

Reply:
`childBlocks` is great 👍 nice!